<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>АРМ Буккроссинг</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbBookList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>481</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string>Списки книг</string>
    </property>
    <widget class="QGroupBox" name="gbISBN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>701</height>
      </rect>
     </property>
     <property name="title">
      <string>Формат: &quot;ISBN&quot; или &quot;Автор заглавие&quot;, без кавычек</string>
     </property>
     <widget class="QTextEdit" name="teISBN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>441</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Книг для поиска:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBooksRemain">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbResultSearch">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>811</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>Результат поиска</string>
    </property>
    <widget class="QStackedWidget" name="swResultPages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>791</width>
       <height>751</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
    </widget>
    <widget class="QLabel" name="lblResultFor">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Результат для:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSearchInfo">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>36</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSendToIRBIS64">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>36</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(238, 238, 238);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
     </property>
     <property name="text">
      <string>Отправить в IRBIS64</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QGroupBox" name="gbSwitchWherePars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Сайты для парсинга</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>88</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ozon.ru</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>88</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>livelib.ru</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>88</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>litres.ru</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: rgb(238, 238, 238);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>1321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Статус выполнения</string>
    </property>
    <widget class="QTextEdit" name="teStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
